packages/flutter_tools/templates/create/lib/main.dart.tmpl

```diff
@@ -84,9 +100,21 @@ class _MyHomePageState extends State<MyHomePage> {
         title: new Text(config.title),
       ),
       body: new Center(
+{{#withPluginHook}}
+        child: new Column(
```

Comment:
My 0.02 is that we should have two templates, a normal app project and a plugin project. The normal project should include driver support, example unit tests, ... - things we think are best practice and that we'd like to lead users towards.